apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -487,10 +491,12 @@ def create_from_matrix(
     try:
         soma_ndarray = cls.open(uri, "w", platform_config=platform_config)
     except DoesNotExistError:
+        # A SparseNDArray must be appendable in soma.io.
+        shape = matrix.shape if cls == DenseNDArray else (None,) * len(matrix.shape)
```

Comment:
`[None for _ in matrix.shape]`, maybe?

Also maybe use `cls.is_sparse` instead of the class check.

(Neither of these are huge issues.)

Reply:
done